src/pcap/cmd/pcap-bosh-cli/main.go

```diff
@@ -298,18 +298,17 @@ func connectToEnvironment(environmentAlias string, config *Config) (*Environment
 			return &environment, nil
 		}
 	}
-	return nil, fmt.Errorf("could not find bosh-environment %s in config", environmentAlias)
+	return nil, fmt.Errorf("could not find bosh-environment %s in BOSH CLI config", environmentAlias)
```

Comment:
we have to check if the whole function is still needed

Reply:
Yes because you still need to connect to the one environment.